home-application/src/main/java/com/softserveinc/ita/homeproject/config/JerseyConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.softserveinc.ita.homeproject.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.ext.ContextResolver;
+import javax.ws.rs.ext.Provider;
+
+@Component
+@ApplicationPath("/api/0")
+public class JerseyConfig extends ResourceConfig {
+
+    public JerseyConfig(ObjectMapper objectMapper) {
+        // register endpoints
+        packages("com.softserveinc.ita.homeproject.api");
+        // register jackson for json
+        register(new ObjectMapperContextResolver(objectMapper));
```

Comment:
Try to use com.fasterxml.jackson.jaxrs.json.JacksonJsonProvider

Reply:
Noted, we will try